src/cluster/mod.rs

```diff
@@ -23,26 +23,27 @@ pub use crate::cluster::pager::{ExecPager, PagerState, QueryPager, SessionPager}
 pub use crate::cluster::rustls_connection_pool::{
     new_rustls_pool, RustlsConnectionPool, RustlsConnectionsManager,
 };
-pub use crate::cluster::session::connect;
+#[cfg(feature = "rust-tls")]
```

Comment:
Shouldn't it be unstable-dynamic-cluster?

Reply:
Yes sorry.